s3client.go

```diff
@@ -165,7 +165,7 @@ func (client *s3client) BucketFileVersions(bucketName string, remotePath string)
 	return versions, nil
 }
 
-func (client *s3client) UploadFile(bucketName string, remotePath string, localPath string, options UploadFileOptions) (string, error) {
+func (client *s3client) UploadFile(bucketName string, remotePath string, localPath string, disableMultipart bool, options UploadFileOptions) (string, error) {
```

Comment:
Please add this as a field to the `UploadFIleOptions` struct instead - that will also reduce the overall number of changes needed for this PR.

Reply:
Good point!
I'll give it a try:+1: